src/block/blog-posts/index.php

```diff
@@ -209,7 +209,7 @@ function stackable_render_blog_posts_block( $attributes, $content ) {
 						$post_id,
 						$attributes['imageSize'],
 						array(
-							'alt' => esc_attr( get_the_title( $post_id ) ),
+							'alt' => esc_attr( get_post_meta( $featured_image_id, '_wp_attachment_image_alt', true ) ),
```

Comment:
@blankart please add a check when the return of `get_post_meta` is `false` or `null`, the `alt` should be an empty string